lib/cli/src/commands/run_unstable.rs

```diff
@@ -129,41 +133,54 @@ impl RunUnstable {
         // without needing to go through and instantiate each one.
 
         let (store, _compiler_type) = self.store.get_store()?;
-        let mut runner = wasmer_wasix::runners::wasi::WasiRunner::new(store)
-            .with_args(self.args.clone())
-            .with_envs(self.wasi.env_vars.clone())
-            .with_mapped_directories(self.wasi.mapped_dirs.clone());
-        if self.wasi.forward_host_env {
-            runner.set_forward_host_env();
-        }
-        if runner.can_run_command(id, command).unwrap_or(false) {
-            return runner.run_cmd(container, id).context("WASI runner failed");
-        }
-
-        let (store, _compiler_type) = self.store.get_store()?;
-        let mut runner = wasmer_wasix::runners::emscripten::EmscriptenRunner::new(store);
-        runner.set_args(self.args.clone());
-        if runner.can_run_command(id, command).unwrap_or(false) {
-            return runner
-                .run_cmd(container, id)
-                .context("Emscripten runner failed");
-        }
+        match command.runner.as_ref() {
```

Comment:
Let's use the constants defined in `webc::metadata:: annotations` instead of string literals. We don't need to worry about things like `runner = "wcgi"` because `wapm2pirita` will automatically convert that to the associated URI.

Reply:
Fixed